components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -290,6 +292,10 @@ class PromptFeature private constructor(
         dismissLoginSelectPrompt()
     }
 
+    override fun onBackPressed(): Boolean {
+        return dismissLoginSelectPrompt()
+    }
```

Comment:
Let's also verify that when a `onBackPressed` is called we dismiss the prompt :)